postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -562,15 +567,16 @@ def _run_and_track_explain(self, dbname, statement, obfuscated_statement, query_
         plan_dict, explain_err_code, err_msg = self._run_explain_safe(
             dbname, statement, obfuscated_statement, query_signature
         )
-        err_tag = "error:explain-{}".format(explain_err_code.value if explain_err_code else None)
-        if err_msg:
-            err_tag = err_tag + "-" + err_msg
-        self._check.count(
-            "dd.postgres.statement_samples.error",
-            1,
-            tags=self._dbtags(dbname, err_tag) + self._check._get_debug_tags(),
-            hostname=self._check.resolved_hostname,
-        )
```

Comment:
Interesting, so we were emitting an error count regardless of if there was an error?

Reply:
Yeah, unfortunately resulting in over-inflated error reports. We could estimate how many metrics were misreported by looking at metrics with the tag `error:explain-None`.